CMakeLists.txt

```diff
@@ -125,12 +125,12 @@ find_package( Python3 COMPONENTS Interpreter REQUIRED )
 ecbuild_add_option( FEATURE FCKIT_VENV
                     DEFAULT OFF
                     DESCRIPTION "Install Python virtual environment with fypp and a yaml parser"
-                    CONDITION Python3_VERSION VERSION_GREATER_EQUAL 3.8 )
+                    CONDITION Python3_VERSION VERSION_GREATER_EQUAL 3.8 AND Python3_VERSION VERSION_LESS 3.12 )
```

Comment:
Unfortunately the ci (i.e. downstream-ci) using various VMs is currently set to cmake 3.18.
Making this decision needs to be synchronised with development section, and is probably not on the order right now.

Reply:
`Added in version 3.19: When a version is requested, it can be specified as a simple value or as a range. For a detailed description of version range usage and capabilities, refer to the [find_package()](https://cmake.org/cmake/help/latest/command/find_package.html#command:find_package) command.`

Yes unfortunately the python version checking support in the `find_package` command was only added in cmake 3.19. I have however made the Python3 check optional, since we only need it for the virtual environment, so the build should not fail if Python is not found as long as the venv is not enabled.